### PR TITLE
added user roles using rolify and user picture in progress

### PR DESCRIPTION
**Pull request by HASNAT204**

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class PagesController < ApplicationController
+  def home
```
Comment (sajidali-dnt) [inline app/controllers/pages_controller.rb:2]: why have you used the method name home instead of index?

**Review by sajidali-dnt — CHANGES_REQUESTED** [review]

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (HASNAT204) [reply app/controllers/pages_controller.rb:2]: i have used home because its home page should i update it to index?

Comment (sajidali-dnt) [reply app/controllers/pages_controller.rb:2]: yes

Comment (HASNAT204) [reply app/controllers/pages_controller.rb:2]: ok

**Pull request merged**